### PR TITLE
Fixes bugs preventing replies from being moved.

### PR DESCRIPTION
**Pull request by 0xjove**
Sadly neither relate to moving threads.

### CODE REVIEW

File: inc/mod/pages.php [hunk 1]
```diff
@@ -1159,7 +1160,7 @@ function mod_move_reply($originBoard, $postID) {
 		openBoard($targetBoard);
 
 		// Find new thread on our target board
-		$query = prepare(sprintf('SELECT thread FROM ``posts_%s`` WHERE `id` = :id', $targetBoard));
+		$query = prepare(sprintf('SELECT thread, id FROM ``posts_%s`` WHERE `id` = :id', $targetBoard));
```
Comment (barrucadu) [inline inc/mod/pages.php]: Am I missing something here, won't the `id` field in the result always just be the bound ID because of the `WHERE` clause?

Comment (0xjove) [inline inc/mod/pages.php]: Yes but the id is needed by the function that $post is passed to. Adding a line like $post['id']=$id; works too. You actually don't need the query at all, the new thread was an argument to the function.

Comment (barrucadu) [inline inc/mod/pages.php]: Where is the `id` needed? it looks like `link_for` doesn't look at the `id` if the `thread` is set.

Comment (0xjove) [inline inc/mod/pages.php]: Thread is null when the post is an OP.

**Pull request merged**